doc/build-windows.md

```diff
@@ -60,7 +60,7 @@ example /usr/src/bitcoin, AND not under /mnt/d/. If this is not the case the dep
 This means you cannot use a directory that is located directly on the host Windows file system to perform the build.
 
 Additional WSL Note: WSL support for [launching Win32 applications](https://learn.microsoft.com/en-us/archive/blogs/wsl/windows-and-ubuntu-interoperability#launching-win32-applications-from-within-wsl)
-results in `Autoconf` configure scripts being able to execute Windows Portable Executable files. This can cause
+results in configure scripts being able to execute Windows Portable Executable files. This can cause
```

Comment:
Was this removed because this is an issue for CMake as well?

Reply:
The documentation has been amended in https://github.com/hebasto/bitcoin/pull/299.

Reply:
Addressed in https://github.com/bitcoin/bitcoin/pull/30454#issuecomment-2271981040.